Makefile

```diff
@@ -72,16 +71,13 @@ populate_data:
 update_code:
 	git checkout master && git pull
 
-update_deps: update_npm_deps
+update_deps:
 	pip install --no-deps --exists-action=w -r requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --find-links https://pyrepo.addons.mozilla.org/ --no-index
 
-update_npm_deps:
```

Comment:
Removing this means the "full install" won't work anymore, right? Might need some documentation about doing that manually?